### PR TITLE
EE-781: Automate unsubscribe from CAC lists.

### PR DESCRIPTION
**Pull request by marklise**
NOBUG: Update language
EE-781: Unsubscribe automatically.
EE-935: Text updates
EE-937: CAC Text changes.
EE-938: Skip intro unltil learn more
EE-925: Add checkbox for understanding terms of reference.
NOBUG: Fix models for EE-936
Text updates

### CODE REVIEW

File: src/app/cac-unsubscribe/cac-unsubscribe.component.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+          <div class="form-group mb-2">
+            <p class="control-label">Project</p>
+            <div class="form-control-container">
+              <label>{{projectName}}</label>
+            </div>
+          </div>
+          <div class="form-group">
+            <p class="control-label">Email Address</p>
+            <div class="form-control-container">
+              <input required email class="form-control" type="text" name="emailInput" id="emailInput" [(ngModel)]="emailInput" autofocus />
```
Comment (KitArmstrong) [inline src/app/cac-unsubscribe/cac-unsubscribe.component.html]: One of our upcoming tickets is to make this more accessible with a screen reader. Is it possible to use a `label` element to better provide screen reader support?

File: src/app/services/project.service.ts [hunk 2]
```diff
@@ -190,4 +190,10 @@ export class ProjectService {
     return this.api.cacSignUp(project, meta)
       .catch(error => this.api.handleError(error));
   }
+
+  // Remvoe this user from the CAC membership on this project
```
Comment (KitArmstrong) [inline src/app/services/project.service.ts]: 'Remove'

File: src/app/cac-unsubscribe/cac-unsubscribe.component.html (hunk 1 above)
Comment (marklise) [reply src/app/cac-unsubscribe/cac-unsubscribe.component.html]: Good catch!  I cleaned it up a bit better including adjusting for label.  Fixed in #4e99353

File: src/app/services/project.service.ts (hunk 2 above)
Comment (marklise) [reply src/app/services/project.service.ts]: Fixed in #4e99353 .

**Pull request merged**